README.md

```diff
@@ -1,33 +1,108 @@
 # Cloud in a Box
 
-**The secrets are stored in plain text and are not secure. Do not use for public
-accessible systems.**
+Cloud in a Box is a minimalistic installation of a OSISM with only services which are needed to make it run.
+For more information you can consult the [Testbed](https://docs.osism.tech/testbed/index.html) documentation.
+If you want to have a full installation please have a look at the [OSISM](https://docs.osism.tech/) documentation.
 
-## Assumptions
+> **Warning:**
+> The secrets are stored in plain text and are not secure. Do not use for public
+> accessible systems.
 
-* the 1st blockdevice is available as ``/dev/sda`` or ``/dev/nvme0n1``
+## Requirements
 
-## Download
+* The first blockdevice is available as `/dev/sda` or `/dev/nvme0n1`.
+* You need to have at least a second blockdevice which is from the same type as the first, empty and usable.
+* USB stick with at least 2GB capacity.
+* CPU: 16 core, 32 threads
+* RAM: 128 GB
+* Storage: 2x 1 TB
+* Network: 2 network interfaces
 
-| :zap: When booting from this image, all data on the hard disks will be destroyed without confirmation. |
-|--------------------------------------------------------------------------------------------------------|
+## Download the Cloud in a Box image
 
-* https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-1.iso (with /dev/sda)
-* https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-2.iso (with /dev/nvme0n1)
+> **Warning:** ⚡ When booting from this image, all data on the hard disks will be destroyed without confirmation.
 
-## Usage
+* <https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-1.iso> (with /dev/sda)
+* <https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-2.iso> (with /dev/nvme0n1)
 
-* Copy image to USB stick
-* Boot from USB stick
-* Installation is performed, system shuts down afterwards
-* Remove USB stick and start system
-* Deloyment is performed, system shuts down afterwards
-* System is ready for use, by default DHCP is tried on
-  the 1st network device
-* Login via ``dragon`` and ``password``
+## Getting started
+
+Download the Cloud in a Box image from the link above.
+
+Use a tool to create a bootable USB stick with the Cloud in a Box image.
+
+Boot from the USB stick.
+
+The instalation will start and take a few minutes. After that the system will shutdown.
+
+Remove the USB stick and restart the system.
+
+The deployment will start. This takes some time and the system will shutdown when the deployment is finished.
+This step can take at least one hour or more.
```

Comment:
This takes roughly an hour, possibly longer depending on the hardware and internet connection.